packages/driver/src/cy/net-stubbing/events/index.ts

```diff
@@ -31,7 +31,9 @@ export function registerEvents (Cypress: Cypress.Cypress, cy: Cypress.cy) {
   const { state } = Cypress
 
   function getRoute (routeId) {
-    return state('routes')[routeId]
+    const routes = state('routes') || {}
+
+    return routes[routeId]
```

Comment:
what was this change for?

Reply:
I was seeing errors in the console for can't read from undefined when running a bunch of files. Not sure which test specifically.